install_msw.sh

```diff
@@ -127,6 +133,38 @@ then
     git config --global core.fileMode false
 fi
 
+if [ "WSL" = "$platform" ]
+then
+    # Install/update packages.
+    packages_list="autoconf automake bsdtar dos2unix doxygen \
+      gdb git libtool make patch pkg-config rsync unzip wget \
+      zip zsh g++-mingw-w64-i686"
+
+    # Disable shellcheck warning: Double quote to prevent globbing and word splitting.
+    # shellcheck disable=SC2086
+    missing_packages_count=$(dpkg-query -W -f="\${Status}\n" $packages_list 2>&1 | \
+      grep -v -c 'install ok installed')
+
+    if [ "$missing_packages_count" -gt 0 ]
+    then
+        sudo apt update
+        # shellcheck disable=SC2086
+        sudo apt install -y $packages_list
+    fi
+
+    # Make symbolic links to lmi and cache_for_lmi directories.
+    # Unfortunately sudo must be used because we don't have permissions to
+    # write to /opt and / directories.
+    mkdir --parents /mnt/c/opt/lmi/src/lmi
+    [ ! -d /opt/lmi ] && \
```

Comment:
I think rewriting this using `if` would be more clear, if only because it would appear better in `set -x` output.

(same thing below, of course).